store/postgres/src/store.rs

```diff
@@ -369,9 +369,10 @@ impl Store {
         }
 
         // Add range filter to query
-        diesel_query = diesel_query
-            .limit(query.range.first as i64)
-            .offset(query.range.skip as i64);
+        if let Some(limit) = query.range.first {
+            diesel_query = diesel_query.limit(limit as i64);
+        }
+        diesel_query = diesel_query.offset(query.range.skip as i64);
```

Comment:
Even though pg seems to optimize `skip == 0` away, it would be clearer to just write the `if skip > 0 { ... }` here explicitly.

Reply:
Done, thanks.